src/testCombiner.ts

```diff
@@ -90,7 +90,20 @@ export function genTestClass(
   const classNameModified = className.replace(/\$/g, '_');
   // Merge all tests' classAnnotation arrays into a single Set
   const classAnnotations = new Set(([] as string[]).concat(...tests.map((test) => test.classAnnotations)));
-  const uniqueTests = tests[0].id ? deduplicateTestNames(tests) : tests;
+  // Standardise test interfaces
+  const standardisedTests = tests.map((t) => {
```

Comment:
This will mutate the contents of `tests`, so there's no need to copy to `standardisedTests`, just use tests as is:
```suggestion
  tests.forEach((t) => {
```